src/styles/react-joyride.css

```diff
@@ -103,7 +103,6 @@ Listing of custom-properties and their default values:
 .joyride-hole {
   border-radius: var(--joyride-hole-border-radius, 4px);
   box-shadow: 0 0 0 9999px var(--joyride-overlay-color, rgba(0, 0, 0, 0.5)), var(--joyride-hole-shadow, 0 0 15px rgba(0, 0, 0, 0.5));
-  position: absolute;
```

Comment:
One annoying thing about what I did in duplicating the styles in this project is that we need to change this in two places.  Here, and in https://github.com/nutshellcrm/react-joyride/blob/4a534e6fb618565ab83512faa6f7071ff68e035e/src/styles/react-joyride.scss#L115